packages/components/pie-button/src/index.ts

```diff
@@ -226,18 +226,22 @@ export class PieButton extends FormControlMixin(LitElement) implements ButtonPro
             responsiveSize,
         } = this;
 
+        const classes = {
+            'o-btn': true,
+            'o-btn--fullWidth': isFullWidth,
+            'is-responsive': isResponsive,
+            [`o-btn--${responsiveSize}`]: Boolean(isResponsive && responsiveSize),
+            [`o-btn--${variant}`]: true,
+            [`o-btn--${size}`]: true,
+            'is-loading': isLoading,
```

Comment:
Maybe we can pass only dynamic class names to the object and combine it with static values like `class="o-btn o-btn--${variant} o-btn--${size} ${classMap(classes)}"`?

Reply:
I think its a reasonable idea, my only argument against this would be it's nice being able to see all the class names in a single place

Reply:
That's pretty much why I ended up using one object for the classes – I preferred being able to see the different classes being set altogether. Would be good to know if people are for or against it though before we refactor in other places 😄